src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2501,6 +2501,50 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/definitions':
```

Comment:
Looking at https://github.com/Alfresco/alfresco-data-model/blob/c7aaa53cce8245889ef79daffac46b53f827ee0e/src/main/java/org/alfresco/service/cmr/dictionary/DictionaryService.java#L137 and https://github.com/Alfresco/alfresco-data-model/blob/c7aaa53cce8245889ef79daffac46b53f827ee0e/src/main/java/org/alfresco/service/cmr/dictionary/ClassDefinition.java I think this might need to be the singular form i.e. /nodes/{nodeId}/definition as we'll probably want to return a single entry object that contains property definitions (to be exposed now) and association definitions (to be exposed later).

Reply:
That's fine but I don't think that the entry object that we will return will have an Id.
Is it fine with you?
In order to identify the respone we could return the typeId and the aspectsId list of the node. What do you think about that?